app/src/main/java/org/lineageos/glimpse/fragments/MediaViewerFragment.kt

```diff
@@ -78,10 +78,24 @@ class MediaViewerFragment : Fragment(R.layout.fragment_media_viewer) {
         viewLifecycleOwner.lifecycleScope.launch {
             mediaUri?.also {
```

Comment:
I'll very much prefer if we switch to if based logic here now, this is getting messy.